### PR TITLE
Update test.py

### PR DESCRIPTION
**Pull request by fadhilmufid**
add new line

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -1 +1,2 @@
 print("Hello World")
+print("bikin branch")
```
Comment (livingmine) [inline test.py]: @fadhilmufid please capitalize the letter to fit our code convention style.

Comment (fadhilmufid) [reply test.py]: @livingmine create new branch again or directly edit in this branch?

Comment (livingmine) [reply test.py]: @fadhilmufid please continue on this branch.

Comment (fadhilmufid) [reply test.py]: @livingmine done

**Review by livingmine — APPROVED** [review]
@fadhilmufid LGTM.

**Pull request merged**